cosmicds/viewers/dotplot/layer_artist.py

```diff
@@ -26,16 +26,23 @@ def __init__(self, view, viewer_state, layer_state=None, layer=None):
             **self.view.scales,
             'rotation': LinearScale(min=0, max=180)
         }
-        self.bars = Scatter(scales=self.scales,
+        
+        if self.view.state.use_bars:
+            self.bars = Bars(
+                scales=self.view.scales, x=[0, 1], y=[0, 1])
+        else:
+            self.bars = Scatter(scales=self.scales,
                             x=[0, 1],
                             y=[0, 1],
                             marker='ellipse')
+            
 
         self.view.figure.marks = list(self.view.figure.marks) + [self.bars]
 
         dlink((self.state, 'color'), (self.bars, 'colors'), lambda x: [color2hex(x)])
         dlink((self.state, 'alpha'), (self.bars, 'opacities'), lambda x: [x])
-        dlink((self.state, 'skew'), (self.bars, 'default_skew'))
+        if  not self.view.state.use_bars:
+            dlink((self.state, 'skew'), (self.bars, 'default_skew'))
```

Comment:
This looks good here, but I think we want to make sure to handle this correctly in `_update_bar_type` as well.

Reply:
Would the right raw to handle this be to remove the link and then recreate it? Is there a way to disable/enable a link in glue?

Reply:
I just relink the skew paramter here 8335b1112be7a269e0a1e6c9981cd49fa43f46a6. I'm not sure if there is some cleanup of the old link that needs to happen, but it seems fine